airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -247,11 +244,11 @@ def execute(self, context: Dict):
         resource = hook.get_query(query_id=self.report_id)
         # Check if report is ready
         if resource["metadata"]["running"]:
-            raise AirflowException('Report {} is still running'.format(self.report_id))
+            raise AirflowException("Report {} is still running".format(self.report_id))
 
         # If no custom report_name provided, use DV360 name
         file_url = resource["metadata"]["googleCloudStoragePathForLatestReport"]
-        report_name = self.report_name or urlparse(file_url).path.split('/')[2]
+        report_name = self.report_name or urlparse(file_url).path.split("/")[2]
```

Comment:
Hm, I think we should use -1 here. Because:
```
In [3]: urlparse("gs://path/to/a/file.txt").path.split("/")[2]
Out[3]: 'a'
```

Reply:
Yep, that's right. I will fix it.

Reply:
Good catch, btw. :)